packages/sqs-json-body-parser/.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "node": "6.10"
```

Comment:
update to node 10

Reply:
While this has merrit I have reservation of updating until core is updated.

Reply:
Is there any plan to make the change across the board to target node 10?